soc/renesas/ra/ra4m1/Kconfig

```diff
@@ -9,3 +9,26 @@ config SOC_SERIES_RA4M1
 	select DYNAMIC_INTERRUPTS
 	select TIMER_READS_ITS_FREQUENCY_AT_RUNTIME
 	select XIP
+
+config SOC_OPTION_SETTING_MEMORY
+	bool "Option Setting Memory"
+	default y
+
+config SOC_OPTION_SETTING_LVDAS_DISABLE
```

Comment:
The voltage detecting and enabling HOCO on reset setting seems good to be managed by hwinfo driver.
If you are having difficulty creating a suitable devicetree at the moment,
I think it's okay to set a fixed value at this point.